docs/user-guide/ze-working-with-commands.md

```diff
@@ -0,0 +1,51 @@
+# Working with commands
+
+You can issue the same commands used in ISPF with Zowe Explorer.
+
+MVS commands can be used to obtain information from the zOS console, while SSH commands can be used to issue shell commands to Unix System Services (USS). TSO commands can be used to invoke mainframe scripts in a zOS terminal session.
+
+## Issuing MVS commands
+
+1. Expand **DATA SETS**, **UNIX SYSTEM SERVICES (USS)**, or **JOBS** in the **Side Bar**.
+2. Right-click on your profile and select the **Issue MVS Command** option.
+
+   Alternatively, press the `F1` key to open the **Command Pallette**, and then select the **Zowe Expolorer: Issue MVS Command** option.
+
+3. In the **Quick Pick**, enter a new command or select a saved command.
+4. Press the `Enter` key to execute the command.
+
+    The output displays in the **Output** panel.
+
+> Example command: Issue `profile` to get information on your zOS system user profile.
```

Comment:
Profile is not a valid MVS command (I think). We could change `profile` to something like `DISPLAY C` or something.

Also, the .gif shows issuing a TSO command.

Reply:
@t1m0thyj  -- can you verify whether the profile command is real?

Reply:
Sorry I think I got the commands flipped. `D T` is a valid MVS command (`display time` is not, it's very picky about abbreviations and casing). And `profile` is a valid TSO command.